packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js

```diff
@@ -62,6 +62,11 @@ class QRCodeCaptureContainer extends React.PureComponent {
       this.props.updateUI({ bchAddress: { toggled: false } })
     } catch (e) {
       try {
+        if (utils.bch.isCashAddr(data)) {
+          this.props.formActions.change('sendBch', 'to', data)
+          this.props.updateUI({bchAddress: {toggled: false}})
+          return
+        }
```

Comment:
Could this be simplified with a `utils.bch.isCashAddr(data) || utils.bitcoin.isValidBitcoinAddress(data)` in the first try? Think Andrew mentioned it last time also